examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
Those changes worked and I've removed the local text shaders from the example.

I did also make it so GLES3 uses the GL3_SHADER path in the Font shader code. For this to work I just had to define a different version for GLES and also make it swap the texture component used from the glyph texture. Hope that's not a step too far, just wanted to properly test gles3. 

The gl2 shader path did work for both gles2 and gles3